.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Task
+about: Add an issue to describe and track an implementation task
+title: ''
+labels: kind/task
+assignees: ''
+
+---
+
+## Description
+
+*A clear and concise description of the functionality that will be implemented
+through completion of this task.*
+
+## Implementation Details
+
+## Definition of Done
```

Comment:
Do we need DoD list here and in the PR template?

Reply:
I don't think so.

Reply:
it's also in the user story